### PR TITLE
add standalone-web xml for jws tests.

### PR DESCRIPTION
**Pull request by gurunrao**
Signed-off-by: gurunrao <gurunandan.rao@oracle.com>
Fix for issue - https://github.com/eclipse-ee4j/jakartaee-tck/issues/881
CI run with tomcat - https://ci.eclipse.org/metro/job/JAXWS_jakarta_TCK_10_run/19/artifact/SUMMARY.TXT
CI run with GF 7 - https://ci.eclipse.org/jakartaee-tck/job/guru/job/jakartaee-tck-build-run/job/jws-webprofile-fix/3/

### CODE REVIEW

**Comment by arjantijms** [comment]
Wasn't the name sun-web.xml replaced by glassFish-web.xml long ago?

**Review by lukasj — APPROVED** [review]
LGTM

@arjantijms where do you see sun-web.xml being introduced?

File: src/com/sun/ts/tests/jws/webmethod/webmethod1/server/standalone.web.xml [hunk 1]
```diff
@@ -18,18 +18,20 @@
...
-    <load-on-startup>0</load-on-startup>
-  </servlet>
-  <servlet-mapping>
-    <servlet-name>@SERVLETNAME@</servlet-name>
-    <url-pattern>@URLALIAS@</url-pattern>
-  </servlet-mapping>
-  <session-config>
+  <display-name>DefaultWebMethodWebServiceApp</display-name>
+  <listener>
+<listener-class>com.sun.xml.ws.transport.http.servlet.WSServletContextListener</listener-class>
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/jws/webmethod/webmethod1/server/standalone.web.xml:23]: Why are we now changing the various @SERVLETNAME@ (and other) parameters?

Comment (lukasj) [reply src/com/sun/ts/tests/jws/webmethod/webmethod1/server/standalone.web.xml:23]: Servlet containers are not required to implement enterprise ws-like deployment (aka jsr-109), see how other tests are written, ie at https://github.com/eclipse-ee4j/jakartaee-tck/tree/master/src/com/sun/ts/tests/jaxws/ee/j2w/document/literal/webmethod/server

Comment (scottmarlow) [reply src/com/sun/ts/tests/jws/webmethod/webmethod1/server/standalone.web.xml:23]: Thanks Lukas!

**Comment by arjantijms** [comment]
> where do you see sun-web.xml being introduced?

In the title of this PR: "add sun-web xml. #883"

**Comment by gurunrao** [comment]
> > where do you see sun-web.xml being introduced?
> 
> In the title of this PR: "add sun-web xml. #883"

renamed the PR to avoid confusion.

**Pull request merged**